velox/row/UnsafeRowDeserializers.h

```diff
@@ -552,6 +552,40 @@ struct UnsafeRowPrimitiveBatchDeserializer {
  * a Vector.
  */
 struct UnsafeRowDeserializer {
+ public:
+  /// Deserialize the rows which is serialized in contiguous memory.
+  /// Deserialize fast when all the column types are primitive type, otherwise,
+  /// deserialize rows one by one, as previously. Fast deserialization
+  /// computes the start memoryAddress of one data by the column index and row
+  /// number, and set all the data of one column at once.
+  /// @param memoryAddress the start memory address of the serialized rows .
+  /// @param type the element type.
+  /// @param offsets offset of each row serialized data. It's size should be
+  /// equal to the deserialized rows. First offset is 0.
+  /// @param pool the memory pool to allocate Vector.
+  static VectorPtr deserialize(
+      const uint8_t* memoryAddress,
+      const RowTypePtr& type,
+      const std::vector<int64_t>& offsets,
+      memory::MemoryPool* pool);
+
+  /**
+   * Deserializes a complex element type to its Vector representation.
+   * @param data A vector of string_view over a given element in the
+   *UnsafeRow.
+   * @param type the element type.
+   * @param pool the memory pool to allocate Vectors from
+   *data to a array.
+   * @return a VectorPtr
+   */
```

Comment:
Not your change, but it would be better to not use the old style /* */ for comments.